README.md

```diff
@@ -36,6 +36,10 @@ The `tree` file holds a JSON encoding of Shift AST for the source text in `f`, w
 The `comments` file holds a JSON encoding of an array of the comments in the source text in `f` in source order, including location information.
 
 
+## Development
+
+The expectations are generated by running `node generate-expectations`, which uses [shift-parser-js](https://github.com/shapesecurity/shift-parser-js). It is expected that you will use an in-development version of that project to update expectations files (using `npm link`), so this project's package.json does not include `shift-parser` even as a dev dependency.
```

Comment:
Be explicit: `npm link /path/to/shift-parser-js`. Put `package.json` in backticks. `shift-parser-js`, not `shift-parser`.

Reply:
Addressed.